packages/@aws-cdk/aws-events/lib/api-destination.ts

```diff
@@ -0,0 +1,130 @@
+import * as crypto from 'crypto';
+import { Duration, Lazy, Names, Resource } from '@aws-cdk/core';
+import { Construct } from 'constructs';
+import { IConnection } from './connection';
+import { CfnApiDestination } from './events.generated';
+
+/**
+ * Supported HTTP operations.
+ */
+export enum HttpMethod {
+
+  /** POST */
+  POST = 'POST',
+  /** GET */
+  GET = 'GET',
+  /** HEAD */
+  HEAD = 'HEAD',
+  /** OPTIONS */
+  OPTIONS = 'OPTIONS',
+  /** PUT */
+  PUT = 'PUT',
+  /** PATCH */
+  PATCH = 'PATCH',
+  /** DELETE */
+  DELETE = 'DELETE',
+
+}
+
+/**
+ * The event Api Destination properties
+ */
+export interface ApiDestinationProps {
+  /**
+   * The ARN of the connection to use for the API destination
+   */
+  readonly connection: IConnection;
+  /**
+   * A description for the API destination.
+   *
+   * @default - none
+   */
+  readonly description?: string;
+  /**
+   * The method to use for the request to the HTTP invocation endpoint.
+   */
+  readonly httpMethod: HttpMethod;
+  /**
+   * The URL to the HTTP invocation endpoint for the API destination..
+   */
+  readonly invocationEndpoint: string;
+  /**
+   * The maximum number of requests per second to send to the HTTP invocation endpoint.
+   * @default - None
+   */
+  readonly invocationRateLimit?: Duration;
```

Comment:
The description says the unit of this value is Hz (Hertz, N per second), so why is its declared type `Duration` ?

Right now I could pass:

```ts
new ApiDestination(..., {
  invocationRule: Duration.hours(1),
})
```

What does that mean?

Reply:
Fixed